pkg/commands/test/command.go

```diff
@@ -138,11 +138,11 @@ func Command() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			summary := &runner.Summary{}
-			if _, err = runner.Run(cfg, configuration.Spec, summary, tests...); err != nil {
+			if _, summary, err := runner.Run(cfg, configuration.Spec, tests...); err != nil {
 				return err
+			} else {
+				printSummary(summary, out)
```

Comment:
Why don't we print summary always

Reply:
if there's an error it means processing failed